pycqed/analysis_v2/timedomain_analysis.py

```diff
@@ -778,7 +816,7 @@ def cal_states_analysis(self):
                             storing_keys, data_mostly_g=data_mostly_g,
                             column_PCA=self.rotation_type == 'column_PCA'))
             else:
-                if len(cal_states_dict) == 3:
+                if len(cal_states_dict) == 3 and 'PCA' not in self.rotation_type:
```

Comment:
Shouldn't self.num_cal_points be used here?

What happens in case of 3 cal points and PCA?

In other instances you convert to lower case when checking for PCA

Reply:
No, we shouldn't use self.num_cal_points because this attribute counts how many calibration segments were used in the experiment. So it will be nonzero even if no calibration point information was provided (reflected by self.cal_states_dict = {qbn: {} for qbn in self.qb_names}. Here we want to know if it should do 3 state rotation, which it should only do if this instruction was given (contained in self.cal_states_dict and self.cal_states_rotations). 

3 cal points and PCA is handled in self.get_cal_states_dict_for_rotation (your question 3 comments above).

Yes, I now tried to standardize this check such that we always convert to .lower().